src/kakarot/instructions/stop_and_arithmetic_operations.cairo

```diff
@@ -434,8 +281,8 @@ namespace Internals {
             return zero_uint;
         }
         let (b_minus_one) = uint256_sub(b, one_uint);
```

Comment:
why do we need to go through this trouble of subbing 1?

also i can't comment on gh because it isn't part of the pr but aren't the above variables wrongly
named?

Reply:
this is old code, not mine, could be optimized for sure, but ouf of the scope of this PR